packages/expo-modules-core/ios/Swift/Convertibles/ConvertibleFromDictionary.swift

```diff
@@ -0,0 +1,60 @@
+
+/**
+ A protocol that allows initializing the object with a dictionary.
+ */
+public protocol ConvertibleFromDictionary: AnyMethodArgument {
+  init()
+  init(dictionary: [AnyHashable : Any?])
+}
+
+/**
+ Provides the default implementation of `ConvertibleFromDictionary` protocol.
+ */
+extension ConvertibleFromDictionary {
+  init(dictionary: [AnyHashable : Any?]) {
+    self.init()
+    let mirror = Mirror(reflecting: self)
+
+    for (label, value) in mirror.children {
+      guard let label = label else {
+        continue
+      }
+
+      let key = label.starts(with: "_") ? String(label.dropFirst()) : label
```

Comment:
Add comment why you're dropping label with `_` prefix 🤔 
I guess it's some internal RN/Expo parameter? 🤔

Reply:
Nope, it's not internal to RN/Expo. These properties are going to be wrapped by a `@propertyWrapper` and if we want to refer to the wrapper instead of the wrapped value we need to use `_`.
So, let's say we have something like this:

```swift
@propertyWrapper
struct ourWrapper {
  let wrappedValue: String {
    didSet {
      wrappedValue = "Hello " + wrappedValue + "!"
    }
  }
}

struct A {
  @ourWrapper
  let property: String = "Barthec"
}

A().property // -> "Hello Barthec!"
A()._property // -> ourWrapper(wrappedValue: "Hello Barthec!")
```